src/vivarium_nih_us_cvd/data/loader.py

```diff
@@ -679,20 +679,43 @@ def is_transition_rate(name: str) -> bool:
 
 def match_rr_to_cause_name(data: Union[str, pd.DataFrame], source_key: EntityKey):
     # Need to make relative risk data match causes in the model
-    map = {
-        "ischemic_heart_disease": [
-            "acute_myocardial_infarction",
-            "post_myocardial_infarction_to_acute_myocardial_infarction",
-        ],
-        "ischemic_stroke": [
-            "acute_ischemic_stroke",
-            "chronic_ischemic_stroke_to_acute_ischemic_stroke",
-        ],
-        "heart_failure": [
-            "heart_failure_from_ischemic_heart_disease",
-            "heart_failure_residual",
-        ],
-    }
+    is_calculated_paf = (
+        (source_key == data_keys.LDL_C.PAF)
+        or (source_key == data_keys.SBP.PAF)
+        or (source_key == data_keys.BMI.PAF)
+    )
```

Comment:
Nit: this can be simplified with:
```suggestion
    is_calculated_paf = source_key in {data_keys.LDL_C.PAF, data_keys.SBP.PAF, data_keys.BMI.PAF}
```